NBXplorer.Client/NBXplorerNetworkProvider.Liquid.cs

```diff
@@ -0,0 +1,77 @@
+﻿using NBitcoin;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using NBitcoin.Altcoins.Elements;
+using NBXplorer.DerivationStrategy;
+using NBXplorer.Models;
+
+namespace NBXplorer
+{
+	public partial class NBXplorerNetworkProvider
+	{
+		public class LiquidExplorerClient : ExplorerClient
+		{
+			public LiquidExplorerClient(NBXplorerNetwork network, Uri serverAddress = null) : base(network, serverAddress)
+			{
+			}
+			public override async Task<TransactionResult> GetTransactionAsync(uint256 txId,
+				CancellationToken cancellation = default)
+			{
+				return await base.GetTransactionAsync<LiquidTransactionResult>(txId, cancellation);
+			}
+		}
+
+		public class  LiquidTransactionResult : TransactionResult
+		{
+			private ElementsTransaction _transaction;
+			public override Transaction Transaction
+			{
+				get => _transaction;
+				set => _transaction = value as ElementsTransaction;
+			}
```

Comment:
this is completely useless.

Reply:
No, without this, the Tx in BTCPay would only be `Transaction` without the required data

Reply:
Nop

Reply:
you were right :)